dbms/src/Storages/Page/V3/BlobStore.cpp

```diff
@@ -115,6 +116,38 @@ void BlobStore::registerPaths()
     }
 }
 
+FileUsageStatistics BlobStore::getFileUsageStatistics() const
+{
+    FileUsageStatistics usage;
+
+    // Get a copy of stats map to avoid the big lock on stats map
+    const auto stats_list = blob_stats.getStats();
```

Comment:
`const auto & stats_list`?

Reply:
* The return type of `getStats` itself is not reference, adding `&` here can not reduce copying
* And we do wang to copy the whole `stats` to avoid locking